dds_web/api/project.py

```diff
@@ -550,7 +550,7 @@ def project_usage(project):
             cost_gbhour = 0.09 / (30 * 24)
 
             # Save file cost to project info and increase total unit cost
-            cost += bhours / 1e9 * cost_gbhour
+            cost = (bhours / 1e9) * cost_gbhour
```

Comment:
I believe this should / can be moved out of the for loop but otherwise looks good 👍🏻